Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -1,22 +1,25 @@
-import React, { createContext, useCallback, useContext, useEffect, useState } from 'react';
-import { without } from 'lodash';
-import { IHaveChildrenProps } from '../components/common/Props';
-import { IContestType } from '../common/types';
-import { ContestState, FilterType, IFilter } from '../common/contest-types';
-import { useHttp } from './use-http';
-import { useUrls } from './use-urls';
-import { generateFilterItems } from '../common/filter-utils';
-
-interface IContestsContext {
+import React, {createContext, useCallback, useContext, useEffect, useState} from 'react';
+import {isEmpty, isNil, without} from 'lodash';
+import {IHaveChildrenProps, IHavePagesProps} from '../components/common/Props';
+import {IIndexContestsType, IPagedResultType} from '../common/types';
+import {ContestState, FilterType, IFilter} from '../common/contest-types';
+import {useHttp} from './use-http';
+import {useUrls} from './use-urls';
+import {generateFilterItems} from '../common/filter-utils';
+import {useLoading} from './use-loading';
+import {useContestStrategyFilters} from "./use-contest-strategy-filters";
+
+interface IContestsContext extends IHavePagesProps {
     state: {
-        contests: IContestType[];
+        contests: IIndexContestsType[];
```

Comment:
`IIndexContestsType`? Is this used in `Index`?

Reply:
Yes. I made an issue for this to be fixed: https://github.com/SoftUni-Internal/exam-systems-issues/issues/157

Reply:
Why not fix it here? It was `IContestType` previously sooo it is obviosly changed in this PR, and should be fixed in this PR